README.md

```diff
@@ -47,9 +47,9 @@ Files matching this pattern will be included in the manifest.
 
 _Default:_ `"**/*.{js,css,png,gif,ico,jpg,map,xml,txt,svg,swf,eot,ttf,woff,woff2}"`
 
-### ignoreFilePattern
+### fileIgnorePattern
 
-Files matching this pattern will _not_ be included in the manifest even if they match filePattern.
+Files matching this pattern will _not_ be included in the manifest even if they match filePattern. And thus will be allways deployed.
```

Comment:
Thanks @bortevik . Nice catch.

In regards to this addition.

> And thus will be allways deployed.

This isn't really necessary. The way the EmberCLI Deploy plugin ecosystem works is that each plugin is responsible for one job and is independent of any other plugins. For instance, the s3 plugin only knows how to copy files to s3. It knows nothing of managing an index file in redis or even the fact it's deploying an application. It simply uploads files to S3.

In the same way, this plugin knows nothing about deployment or uploading files. All it does is simply create a manifest file based on a collection of files that match the `filePattern` property.

Therefore there is nothing to say that the files in the `fileIgnorePattern` won't be deployed anywhere because it's not up to this plugin. It's up to any plugin responsible or deploying files and whether they respect what is in the manifest.

So, I'd suggest removing this additional line as it simply is not relevant to this plugin.